bindings/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2020 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+option(FRAMEWORK_USES_PYTHON "Do you want to create the Python bindings" FALSE)
+
+if(FRAMEWORK_USES_PYTHON)
+
+    # Find the necessary Python libraries
+    find_package(Python3 COMPONENTS Interpreter Development REQUIRED)
+    message(STATUS "Using Python: ${Python3_EXECUTABLE}")
+
+    # Install the resulting Python package for the active interpreter
+    if(NOT PYTHON_INSTDIR)
+        set(PYTHON_INSTDIR ${Python3_SITELIB}/bipedal_locomotion)
+    endif()
+
+    # Find pybind11
+    find_package(pybind11 CONFIG REQUIRED)
+
+    # Folder of the Python package within the build tree.
+    # It is used for the Python tests.
+    set(BIPEDAL_LOCOMOTION_PYTHON_PACKAGE "${CMAKE_BINARY_DIR}/bipedal_locomotion")
```

Comment:
Just to understand, what will be the package name?  The actual name of the bundle is BipedalLocomotionFramework. To include it all at once you can do
```
#include <BipedalLocomotion/Framework.h>
```
Then, to match also the file structure, the namespace is shortened in ``BipedalLocomotion``, so that we have ``BipedalLocomotion::Planners``, ``BipedalLocomotion::Estimators``, and eventually ``BipedalLocomotion::Controllers``. That's why the namespace is shortened, but only to have good looking target names :grin:

Reply:
@GiulioRomualdi in #131 used `import BipedalLocomotion`, but since it's not compliant with PEP8 as I mentioned in https://github.com/dic-iit/bipedal-locomotion-framework/pull/131#issuecomment-714577349 I switched to `import bipedal_locomotion`. If you prefer, the package name could become `import bipedal_locomotion_framework`, it'd be equally fine for me.

Reply:
Ah ok. I missed that. I would prefer ``bipedal_locomotion_framework``, but I would leave the final decision to @GiulioRomualdi as it developed them in the first place.

Reply:
Personally I like more the complete name, since I got used to use `blf` as a shorthand for the project, and I'd like to do:

```python
import bipedal_locomotion_framework.bindings as blf
```

but it's your choice :)

Reply:
Ok, I agree!

Reply:
I vote for `import bipedal_locomotion_framework`

Reply:
Done in 5b00071